nni/experiment/__init__.py

```diff
@@ -2,5 +2,5 @@
 # Licensed under the MIT license.
 
 from .config import *
-from .experiment import Experiment
-from .data import *
+#from .experiment import Experiment
```

Comment:
remove the line if it is not used anymore, or add annotation to annotate why it should keep.

Reply:
It will be added back in following stages. It's temporarily removed to run UT for the new module, or otherwise the test case will raise import error.